InterGalacticCurrencyConvertor/src/main/java/DataProcessResult.java

```diff
@@ -0,0 +1,44 @@
+package main.java;
+
+public class DataProcessResult{
```

Comment:
Consider making this class immutable i.e. the member variables become final and you set the members in the c'tor.

Reply:
updated